### PR TITLE
pull_request

### PR DESCRIPTION
**Pull request by navan62**

### CODE REVIEW

File: task_js/index.js [hunk 1]
```diff
@@ -62,3 +62,5 @@ function setTasks(tasks) {
     localStorage.setItem("tasks", JSON.stringify(tasks));
```
Comment (galamo) [inline task_js/index.js:62]: Hi
the pull request should contain all the project files
so i will be able to comment on all files

couldent do it

Comment (navan62) [reply task_js/index.js:62]: OK,
I upload the files!
sorry i was late.

Comment (galamo) [reply task_js/index.js:62]: Still not good!

**Comment by galamo** [comment]
Again your project was not uploaded.

your grade is 98

please use Classes instead inline CSS

Good luck


‫בתאריך יום א׳, 23 בינו׳ 2022 ב-0:55 מאת ‪navan62‬‏ <‪
***@***.***‬‏>:‬

**Pull request merged**